test/env/array/array-env.test.js

```diff
@@ -17,10 +17,12 @@ describe('env array', () => {
         expect(stderr).toBeFalsy();
         expect(stdout).toBeTruthy();
 
-        const devScript = spawnSync('node', [devFile]);
-        const prodScript = spawnSync('node', [prodFile]);
+        if (isWebpack5) {
```

Comment:
then how was it working on webpack 4 before?

Reply:
I'm still investigating the root cause :dissappointed:

Reply:
yep investigate investigate